src/Microsoft.Diagnostics.Monitoring.RestServer/EventStreamResult.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    /// <summary>
+    /// Same as FileStreamResult, but also cleans up the underlying stream provider once it's finished
```

Comment:
????? https://github.com/dotnet/aspnetcore/blob/328dd0ac8a3100e93eaceac1a832f8349fea5fda/src/Mvc/Mvc.Core/src/Infrastructure/FileResultExecutorBase.cs#L364

Why do you need this result type?